server/plugin/api.go

```diff
@@ -27,8 +27,25 @@ const (
 	// TokenTTL is the OAuth token expiry duration in seconds
 	TokenTTL = 10 * 60
 
-	requestTimeout       = 30 * time.Second
-	oauthCompleteTimeout = 2 * time.Minute
+	requestTimeout         = 30 * time.Second
+	oauthCompleteTimeout   = 2 * time.Minute
+	HeaderMattermostUserID = "Mattermost-User-ID"
+	ownerQueryParam        = "owner"
+	repoQueryParam         = "repo"
+	numberQueryParam       = "number"
+	postIdQueryParam       = "postId"
+
+	IssueStatus         = "status"
+	AssigneesForProps   = "assignees"
+	LabelsForProps      = "labels"
+	DescriptionForProps = "description"
+	TitleForProps       = "title"
+	IssueNumberForProps = "issue_number"
+	IssueUrlForProps    = "issue_url"
+	RepoOwnerForProps   = "repo_owner"
+	RepoNameForProps    = "repo_name"
```

Comment:
Its better to group the constants like props and params etc
```suggestion
	RepoNameForProps    = "repo_name"

```